microservices-connector/README.md

```diff
@@ -1,4 +1,4 @@
-# genai-microservices-connector(GMC)
+# genai-microservices-connector(GMC)T
```

Comment:
LGTM please take care of this later

Reply:
sure. It has been removed the new commit.